docs/_quarto.yml

```diff
@@ -78,6 +78,7 @@ website:
             - reference/node/linear-resistance.qmd
             - reference/node/manning-resistance.qmd
             - reference/node/terminal.qmd
+            - reference/node/junction.qmd
```

Comment:
The tutorials should also be switched over to use a Junction node. Right now they use a Basin called confluence, see https://ribasim.org/tutorial/natural-flow.html#basin-node-confluence.

We could keep the name "confluence" but make the Basin a Junction instead.

Reply:
I've made an issue for this, as we determined that we cannot easily change the existing models (they need basins).